packages/components/organisms/f-checkout/src/services/tests/addressService.test.js

```diff
@@ -174,4 +175,101 @@ describe('addressService', () => {
             });
         });
     });
+
+    describe('isAddressInLocalStorage ::', () => {
+        describe('if localStorage exists', () => {
+            beforeEach(() => {
+                Object.defineProperty(window, 'localStorage', { value: localStorageMock });
+            });
+
+            afterEach(() => {
+                window.localStorage.clear();
+                jest.resetAllMocks();
+            });
+
+            describe('when the address does NOT exist in local storage', () => {
+                it('should return false', () => {
+                    expect(addressService.isAddressInLocalStorage()).toBe(false);
+                });
+            });
+
+            describe('when the address does exist in local storage', () => {
+                it('should return true if there is a postcode', async () => {
+                    // Arrange & Act
+                    window.localStorage.setItem('je-full-address-details', JSON.stringify(london3LinesDefault));
+
+                    // Assert
```

Comment:
Going to be picky :P the Act is actually this part. So here it should be `// Act & Assert`